### PR TITLE
Update READMEs - L02-04

### PR DESCRIPTION
**Pull request by endgame**

### CODE REVIEW

File: level02/README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+without our machine exploding. Because of how we've built these `Command`s, any
+sequence of random actions is valid. This is only okay when our actions can run
+regardless of the current state of the system.
 
-What about when we have `Command`s that do not make sense to run unless the
-system has reached a particular state? If we do not manage this, we are re left
-with a non-deterministic test suite because some `Command`s _might_ be generated
-in an absurd order.
+What about `Command`s that only make sense when the system has reached a
+particular state? If we do not manage this, we are re left with a
```
Comment (mankyKitty) [inline level02/README.md]: `we are re left`, typo me thinks

File: level03/README.md [hunk 2]
```diff
@@ -27,9 +27,8 @@ We can add either milk or sugar with the following functions:
...
 ```haskell
 addMilk :: MonadIO m => Machine -> m ()
---
 addSugar :: MonadIO m => Machine -> m ()
 ```
 
-You may implement this as one or two commands. As these functions have a lot in
-common, we recommend implementing this as a single `Command`.
+You may implement this as one or two commands. As these functions have
+a lot in common, try doing it as a single `Command`.
```
Comment (mankyKitty) [inline level03/README.md:34]: yes it is

File: level02/README.md (hunk 1 above)
Comment (endgame) [reply level02/README.md]: arrrrrrrrrr

File: level03/README.md (hunk 2 above)
Comment (endgame) [reply level03/README.md:34]: you're shitposting, right?

Comment (mankyKitty) [reply level03/README.md:34]: HAHAHA :1st_place_medal: <3

Comment (endgame) [reply level03/README.md:34]: o7

**Pull request merged**